addons/core/package.json

```diff
@@ -50,45 +51,45 @@
   },
   "devDependencies": {
     "@babel/core": "^7.26.10",
-    "@babel/eslint-parser": "^7.21.3",
-    "@babel/plugin-proposal-decorators": "^7.21.0",
-    "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.0.1",
-    "@ember/test-helpers": "^4.0.5",
-    "@embroider/test-setup": "^2.1.1",
+    "@babel/eslint-parser": "^7.25.1",
+    "@babel/plugin-proposal-decorators": "^7.24.7",
+    "@ember/optional-features": "^2.1.0",
+    "@ember/test-helpers": "^3.3.1",
+    "@embroider/test-setup": "^4.0.0",
     "@faker-js/faker": "^8.0.2",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "broccoli-asset-rev": "^3.0.0",
     "concurrently": "^9.1.0",
     "doctoc": "^2.2.0",
     "ember-cli": "^5.12.0",
```

Comment:
when running the cmd to update ember it also suggested to change `"ember-cli": "^5.12.0",` to `"ember-cli": "~5.12.0",`. Should we go with using `~` instead to follow the pattern suggested by ember?

Reply:
true https://github.com/ember-cli/ember-addon-output/blob/v5.12.0/package.json#L46

Reply:
100% this was my bad, it actually changed to `~` but on my pursue of solving the issue with `ember-test-helpers` I had yesterday, I revert that change and forgot to clean it up.
Thank you for the catch!! 💪